go-controller/pkg/ovn/policy.go

```diff
@@ -146,7 +146,7 @@ func (oc *Controller) syncNetworkPolicies(networkPolicies []interface{}) {
 	if len(stalePGs) > 0 {
 		err = libovsdbops.DeletePortGroups(oc.nbClient, stalePGs...)
 		if err != nil {
-			klog.Errorf("Error removing stale port groups %v: %v", stalePGs, err)
+			klog.Fatalf("Error removing stale port groups %v: %v", stalePGs, err)
```

Comment:
line 138 and 143 should also be klog.Fatalf

Reply:
ack. fixed.